app/src/test/java/org/pesmypetcare/mypetcare/features/pets/TestPetUpdate.java

```diff
@@ -0,0 +1,110 @@
+package org.pesmypetcare.mypetcare.features.pets;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.pesmypetcare.mypetcare.controllers.TrUpdatePet;
+import org.pesmypetcare.mypetcare.features.users.User;
+import org.pesmypetcare.mypetcare.services.StubPetManagerService;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class TestPetUpdate {
+    private Pet pet;
+    private User usr;
+    private TrUpdatePet trUpdatePet;
+    private final String name = "Manolo";
+    private final String husky = "Husky";
+    private final String bulldog = "Bulldog";
+    private final String linuxName = "Linux";
+    private final int number30 = 30;
+    private final int number20 = 20;
+
+
+    @Before
+    public void setUp() throws PetRepeatException {
+        usr = new User("johnDoe", "johndoe@gmail.com", "1234");
+        pet = new Pet();
+        pet.setName(name);
+        pet.setGender(Gender.FEMALE);
+        pet.setBirthDate("2 MAR 2010");
+        pet.setBreed(husky);
+        pet.setRecommendedDailyKiloCalories(2);
+        pet.setWashFrequency(2);
+        pet.setWeight(2);
+        trUpdatePet = new TrUpdatePet(new StubPetManagerService());
+    }
+
+    @Test
+    public void shouldChangeGender() {
+        pet.setGender(Gender.MALE);
+        assertEquals("Correct gender change", Gender.MALE, pet.getGender());
+    }
+
+    @Test
+    public void shouldChangeWeight() {
+        pet.setWeight(number30);
+        assertEquals("Correct weight change", number30, pet.getWeight(), 0);
+    }
+
+    @Test
+    public void shouldChangeName() throws PetRepeatException {
+        pet.setName("Ohio");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "Ohio" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5219990)